recipes/findspark/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: findspark
+  version: "1.0.0"
+
+source:
+  fn: findspark-1.0.0.tar.gz
```

Comment:
You can  use `jinja` variables to propagate the version. Just add `{% set version = "1.0.0" %}` to the top of the file and then change the file name to findspark-{{ version }}.tar.gz